### ISSUE TITLE
Lots of vue errors in Timeline view

### ISSUE THREAD

**Issue by johan-bjareholt** [ActivityWatch/aw-webui#191]
Not sure if it's something new or old.
View still works as it should however, there's just something strange going on with vuex I believe

It spams the following

```
[Vue warn]: Error in callback for watcher "function () { return this._data.$$state }": "Error: [vuex] do not mutate vuex store state outside mutation handlers."
Error: "[vuex] do not mutate vuex store state outside mutation handlers."
```

and then ends with

```
WARNING: infinite loop in redraw?
```

**Comment by ErikBjare** [comment]
I think `WARNING: infinite loop in redraw?` is related to vis-timeline, but that shouldn't be in the Stopwatch view?

**Comment by johan-bjareholt** [comment]
@ErikBjare Sorry, not stopwatch but in timeline view. Fixing the title

**Comment by johan-bjareholt** [comment]
I'm ignoring the vis-timeline infinite loop redraw error and consider this to be fixed by #208 which was a vuex error on our end.
